scripts/ceph-windows/setup_libvirt

```diff
@@ -74,12 +74,13 @@ function get_libvirt_vm_ssh_address() {
 #
 # Setup requirements (if needed)
 #
+sudo apt-get update
+sudo apt-get install -y libvirt-daemon-system
```

Comment:
Is there any downside to also specifying `virtinst` and `cloud-image-utils` here and getting rid of the if statements below entirely?

Reply:
Good point, right away.